pixi/src/ui/report/CoreWebVitalsReportView.js

```diff
@@ -116,13 +63,15 @@ class CoreWebVitalView {
   render(report) {
     const {data, unit} = report;
 
-    this.scale.render(data, unit);
+    this.scale.render(data);
 
     const responseCategory = data.category.toLowerCase();
     const displayCategory = CATEGORIES[responseCategory];
     this.container.classList.add(responseCategory);
     this.category.textContent = displayCategory;
 
+    const average = (data.numericValue / unit.conversion).toFixed(2);
```

Comment:
1 decimal place should be enough :)

Reply:
Okay! Even for CLS ? :)

Reply:
Good point, no, we do want 2 for CLS. However for FID we want an integer ms result. Let's handle this in another PR.